app/models/shipit/deploy.rb

```diff
@@ -22,7 +22,7 @@ def build_rollback(user = nil, env: nil)
         parent_id: id,
         since_commit: stack.last_deployed_commit,
         until_commit: until_commit,
-        env: env || {},
+        env: filter_deploy_envs(env || {}),
```

Comment:
Let's keep rollbacks out for now. I need to think about it. There is a case for rollbacks re-using the env of the deploy they rollback to. But not 100% sure yet.